server/UserInfoClasses.ts

```diff
@@ -0,0 +1,70 @@
+/******************************************************************************/
```

Comment:
smilar justification as client. decouple expected input and out of server sockets to increase robustness and ease developers having to remember and merge the client and server apis. this serves as an API for the server